src/ICeWall/doc/set_bump.md

```diff
@@ -0,0 +1,38 @@
+## set_bump_options
+
+### Synopsis
+```
+  % set_bump \
+        -row row \
+        -col col \
+        [(-power|-ground|-net) net_name] \
```

Comment:
It might be good to also have an option to leave a bump "unassigned". The bump might still need to be there, but not assigned or connected to anything (such as the power grid) to anything. `-unassigned` (or we could just assign it to a dummy net not connected to anything)

Reply:
Created JIRA ticket OR1054 for this issue